src/ctim/examples/vulnerabilities.cljc

```diff
@@ -123,3 +123,6 @@
 
 (def new-vulnerability-minimal
   {:description "Improper Neutralization of Directives in Dynamically Evaluated Code ('Eval Injection')"})
+
+(def vulnerability-maximal-v3p1
+  (assoc-in vulnerability-maximal [:impact :cvss_v3 :vector_string] "CVSS:3.1/AV:L/AC:L/PR:H/UI:N/S:U/C:H/I:H/A:H"))
```

Comment:
We generally just provide a minimal and maximal example, but do not add other examples for different values.

Reply:
Can do - thanks!